tools/protobuf-compiler/src/main.rs

```diff
@@ -37,8 +37,7 @@ fn main() {
     cmd.arg("build")
         .arg("--output")
         .arg(buf_img.path())
-        .arg("--as-file-descriptor-set")
-        .arg(&src_dir);
```

Comment:
Indeed not necessary? Or does this now require that the command is always run from the repo root? How does this interact with `find_protos` further down (line 60/61)?

Reply:
Seems to not be, workspace seems to solve. I've tested by adding changes and generating to make sure they show up. Find protos still needs the src dir, the workspace features maps things correctly